lib/charms/grafana_k8s/v0/grafana_dashboard.py

```diff
@@ -219,7 +219,7 @@ def __init__(self, *args):
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
 
-LIBPATCH = 32
+LIBPATCH = 33
```

Comment:
Next time, I suggest updating the libs in a separate PR so that PRs are more atomic and changes can be reverted more easily